sdk/core/azure-core-amqp/src/impl/rust_amqp/rust_amqp/azure_core_amqp/Test-Cleanup.ps1

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# cspell: ignore JOBID
+
+param (
+  [string]$PackageName
+)
+
+. "$PSScriptRoot\..\..\..\eng\common\scripts\common.ps1"
+
+if (-not $PackageName) {
+  Write-Host "Please provide a package name."
+  exit 1
+}
+
+
+Write-Host "Test Broker output:"
+Receive-Job -Id $($env:TEST_BROKER_JOBID)
```

Comment:
Nit: unnecessary complexity since you're not in a string. THis actually creates a child scope which does unnecessary processing.
```suggestion
Receive-Job -Id $env:TEST_BROKER_JOBID)
```